scss/_base_code.scss

```diff
@@ -23,9 +24,16 @@ $code-sidebar-width: calc(#{$sph-inner} + (4 * #{$digit-width})) !default;
   code,
   kbd,
   samp {
-    background-color: $color-mid-x-light;
-    box-decoration-break: clone;
-    padding: $sph-inner--x-small;
+    background-color: $color-code-background;
+    border-radius: $border-radius;
+    box-decoration-break: slice;
+    color: inherit;
+    line-height: map-get($line-heights, default-text) - $code-inline-padding;
+    padding: calc(#{$code-inline-padding} - 1px) $code-inline-padding;
+
+    .is-dark & {
```

Comment:
Default text color change happens in `p-strip--dark` class, so it should work with that, not `is-dark`. Same as icons currently support `p-strip--dark`.